eng/common/scripts/get-codeowners.ps1

```diff
@@ -14,30 +20,105 @@ if (!(Test-Path $codeOwnersLocation)) {
 
 $codeOwnersContent = Get-Content $codeOwnersLocation
 
+function VerifyAlias($APIUrl)
+{
+  if ($AuthToken) {
+    $headers = @{
+      Authorization = "bearer $AuthToken"
+    }
+  }
+  try
+  {
+    $response = Invoke-RestMethod -Headers $headers $APIUrl
+  }
+  catch 
+  {
+    Write-Host "Invoke-RestMethod ${APIUrl} failed with exception:`n$_"
+    Write-Host "This might be because a team alias was used for user API request or vice versa."
+    return $false
+  }
+  return $true
+}
+
 foreach ($contentLine in $codeOwnersContent) {
   if (-not $contentLine.StartsWith("#") -and $contentLine){
     $splitLine = $contentLine -split "\s+"
     
     # CODEOWNERS file can also have labels present after the owner aliases
     # gh aliases start with @ in codeowners. don't pass on to API calls
-    $ownedFolders[$splitLine[0].ToLower().Trim("/")] = ($splitLine[1..$($splitLine.Length)] `
-      | ? { $_.StartsWith("@") } `
-      | % { return $_.substring(1) }) -join ","
+
+    $aliases = ($splitLine[1..$($splitLine.Length)] | ? { $_.StartsWith("@") } | % { return $_.substring(1) }) -join ","
```

Comment:
Given that you split on "," below we should just avoid the join here and just keep an array.